components/StyledFilters.js

```diff
@@ -9,12 +9,13 @@ import { Span } from './Text';
  */
 const StyledFilters = ({ filters, getLabel, onChange, selected, minButtonWidth, ...flexProps }) => {
   return (
-    <Flex css={{ overflowX: 'auto' }} {...flexProps}>
+    <Flex data-cy="filters" css={{ overflowX: 'auto' }} {...flexProps}>
       {filters.map((filter, idx) => {
         const isSelected = filter === selected;
         return (
           <StyledButton
             data-cy="filter-button"
+            data-type={`${filter.toLowerCase()}`}
```

Comment:
The reason why we use `data-cy` is that it gets stripped out of the attributes during the build so that it's not included in production code. We should stick with `data-cy` as much as possible and remove this `data-type` if we can avoid it.